app/Model/ComposerConfiguration.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace ArtifactCreation\Model;
+
+/**
+ * Configuration object used to execute composer commands
+ *
+ * Class ComposerConfiguration
+ * @package ArtifactCreation\Model
+ */
+class ComposerConfiguration extends ConfigurationAbstract
+{
+    /**
+     * Set the command
+     */
+    protected function setCommand()
+    {
+        $command = 'composer install --no-dev --ignore-platform-reqs --no-interaction';
```

Comment:
Shouldn't flags of composer e.g. [--no-dev --ignore-platform-reqs --no-interaction] parametrised?